src/routes/user/index.js

```diff
@@ -1,5 +1,5 @@
 import { HTTP_SUCCESS, HTTP_CREATED, HTTP_ERROR, } from 'config'; 
-import { successfulResponse } from '../../util/response';
+import { createSuccessfulResponse } from '../../util/response';
```

Comment:
We also want to create Error Responses. We want the responses to have a message, error code, data, and success. Messages should not be templated since there could be many different messages that error out due to db problems, logic, input, etc. So, take in a message, and error code for this function. The data will always be an empty object.